main/init/sbt/test/resources/expected/config/sbt-multi-project-example/build.mill

```diff
@@ -0,0 +1,155 @@
+package build
+
+import mill._
+import mill.javalib._
+import mill.javalib.publish._
+import mill.scalalib.SbtModule
+
+object Deps {
+
+  val `ch.qos.logback:logback-classic` =
+    ivy"ch.qos.logback:logback-classic:1.2.3"
+
+  val `com.github.julien-truffaut:monocle-core` =
+    ivy"com.github.julien-truffaut::monocle-core:1.4.0"
+
+  val `com.github.julien-truffaut:monocle-macro` =
+    ivy"com.github.julien-truffaut::monocle-macro:1.4.0"
+
+  val `com.github.pureconfig:pureconfig` =
+    ivy"com.github.pureconfig::pureconfig:0.8.0"
+
+  val `com.typesafe.akka:akka-stream` =
+    ivy"com.typesafe.akka::akka-stream:2.5.6"
+
+  val `com.typesafe.scala-logging:scala-logging` =
+    ivy"com.typesafe.scala-logging::scala-logging:3.7.2"
+  val `com.typesafe:config` = ivy"com.typesafe:config:1.3.1"
+
+  val `net.logstash.logback:logstash-logback-encoder` =
+    ivy"net.logstash.logback:logstash-logback-encoder:4.11"
+  val `org.scalacheck:scalacheck` = ivy"org.scalacheck::scalacheck:1.13.5"
+  val `org.scalatest:scalatest` = ivy"org.scalatest::scalatest:3.0.4"
+  val `org.slf4j:jcl-over-slf4j` = ivy"org.slf4j:jcl-over-slf4j:1.7.25"
+}
+
+object `package` extends RootModule with BaseModule {
+
+  def artifactName = "sbt-multi-project-example"
+
+  object common extends BaseModule {
+
+    def ivyDeps = super.ivyDeps() ++ Seq(
+      Deps.`ch.qos.logback:logback-classic`,
+      Deps.`com.typesafe.akka:akka-stream`,
+      Deps.`com.typesafe.scala-logging:scala-logging`,
+      Deps.`com.typesafe:config`,
+      Deps.`net.logstash.logback:logstash-logback-encoder`,
+      Deps.`org.slf4j:jcl-over-slf4j`
+    )
+
+    object tests extends MavenTests with TestModule.ScalaTest {
```

Comment:
Why is this `MavenTests` and not `SbtTests`?

Reply:
Sorry I think I missed this in common buildgen code.

Reply:
Fixed.

https://github.com/com-lihaoyi/mill/blob/1d8011cec7464e1dc87136e29ca4ed8c07fa0451/main/init/sbt/test/resources/expected/config/without-base-project/sbt-multi-project-example/build.mill#L77